pagopa/io-app
# chore(IT Wallet): [SIW-1128] Add `EidCard` and `EidCardPreview` components

## Short description
This PR introduces the `EidCard` and `EidCardPreview` component for the IT Wallet EId credential 

## List of changes proposed in this pull request
- Added `EidCard` component
- Added `EidCardPreview` component
- Added tests
- Added localization keys

## How to test
Go to **Profile > Design System > Cards** and scroll to the bottom of the screen. Check that the components are rendered correctly

## Preview

https://github.com/pagopa/io-app/assets/6160324/c5d9bcae-aa14-41eb-ab8c-6062468c48bc
